membership-attribute-service/app/services/PaymentFailureAlerter.scala

```diff
@@ -125,4 +127,28 @@ object PaymentFailureAlerter extends LoggingWithLogstashFields {
 
     alertAvailable.getOrElse(Future.successful(false))
   }
+
+  def mostRecentPayableInvoicesOlderThanOneMonth(recentInvoices: List[Invoice]): List[Invoice] = {
```

Comment:
why is "payable" part of this name ? is it because they are non zero invoices ?

Reply:
Yes, unfortunately Zuora will still raise an invoice even though the amount is 0 (e.g. if there is 100% discount), so I wanted to make it explicit that such invoices will be ignored